src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/HttpContextExtensions.cs

```diff
@@ -25,6 +25,16 @@ public static SummaryType GetSummaryTypeOrDefault(this HttpContext context)
             {
                 return summary;
             }
+            else if (string.IsNullOrWhiteSpace(query))
+            {
+                var res = context.Request.Query[KnownQueryParameterNames.Count].FirstOrDefault();
+
+                if (!string.IsNullOrWhiteSpace(res) && Convert.ToInt32(res) == 0 &&
```

Comment:
Use a TryParse since this is user input and might not be valid

Reply:
Done . Used TryParse.